jailer/src/cgroup.rs

```diff
@@ -250,35 +252,38 @@ mod tests {
         // 1. If parent file does not exist, return an error.
 
         // This is /A/B/ .
-        let dir = tempdir().expect("Cannot create temporary directory.");
+        let dir = TempDir::new("/tmp/").expect("Cannot create temporary directory.");
         // This is /A/B/C .
-        let dir2 = tempdir_in(dir.path()).expect("Cannot create temporary directory.");
-        let mut path2 = PathBuf::from(dir2.path());
+        println!("{:?}", dir.as_path().join("X"));
```

Comment:
This `print` is a leftover I guess.

Reply:
Yes. Removed it. Take a look: [jailer/src/cgroup.rs:257](https://github.com/firecracker-microvm/firecracker/blob/87231aa9c2462d00b58edf264d1a25482696281a/jailer/src/cgroup.rs#L257)